src/ai/backend/agent/stats.py

```diff
@@ -93,7 +95,7 @@ async def get_preferred_mode():
         return StatModes.DOCKER
 
     @classmethod
-    async def check_docker_cgroup_driver(cls):
+    async def check_docker_cgroup_driver(cls) -> StatModes:
```

Comment:
I think this should be `Optional` as the method may return `None`.

Reply:
Hmm? I alreday applied Optional type in this commit 44bb4cf

Reply:
Ah, my mistake. I just checked the first commit only; Thanks!